app/assets/javascripts/modules/coronavirus-local-restrictions.js

```diff
@@ -0,0 +1,37 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function CoronavirusLocalRestrictions () {}
```

Comment:
Similarly here, I suggest `CoronavirusTrackLocalRestrictions` for the reason mentioned above.

Reply:
Done